lms/templates/courseware/course_updates.html

```diff
@@ -0,0 +1,29 @@
+<%! from django.utils.translation import ugettext as _ %>
+<section>
+<div class="recent-updates">
+  % for index, update in enumerate(visible_updates):
+    <article>
```

Comment:
A few notes related to this: 
1.) it would be great to add a class to this here, and then apply the following styles specific to just these articles. (border-radius:3px; background-color: $white border:1px solid transparent)

2.) On hover, we should also add a border: 1px solid $gray-l3;

3.) On hover for a collapsed post the entire element should be clickable instead of just the "Show" text label. If this item is enough effort to put into a separate ticket please create that ticket and feel free to ignore #3 for the purposes of this PR.